components/myforms/Tabs/TabsList.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef, useState } from "react";
```

Comment:
Originally Tabs.tsx. 
(mentioned above about deleting Tabs and recreating as TabsList)